packages/portfolio-contract/src/constants.js

```diff
@@ -21,3 +21,22 @@ export const AxelarChains = /** @type {const} */ ({
   Base: 'Base',
 });
 harden(AxelarChains);
+
+/**
+ * Strategies for portfolio rebalancing of bulk deposits.
+ *
+ * @enum {(typeof RebalanceStrategy)[keyof typeof RebalanceStrategy]}
+ */
+export const RebalanceStrategy = /** @type {const} */ ({
+  /**
+   * Do what I mean, i.e., use the default strategy specified by the portfolio
+   * configuration.
+   */
+  DoWhatIMean: 'dwim',
```

Comment:
I think it's less "do what I (the sender) mean" as "do what you have already been told to do". Maybe `current`?

A bot offers:
 - configured
 - stored
 - preset
 - baseline

I kinda like preset